### PR TITLE
[SymbolGraph] Only include where clause constraints in `swiftExtension`

### PR DESCRIPTION
**Pull request by bitjammer**
Otherwise, this creates noise in conditional conformance phrasings.

rdar://64425199

### CODE REVIEW

**Comment by bitjammer** [comment]
@swift-ci Please smoke test

File: lib/SymbolGraphGen/Edge.cpp [hunk 1]
```diff
@@ -45,8 +45,8 @@ void Edge::serialize(llvm::json::OStream &OS) const {
       SmallVector<Requirement, 4> FilteredRequirements;
       filterGenericRequirements(
           ConformanceExtension->getGenericRequirements(),
-          ConformanceExtension->getExtendedNominal()
-              ->getDeclContext()->getSelfNominalTypeDecl(),
+                                ConformanceExtension->getExtendedNominal()
+                                  ->getDeclContext()->getSelfNominalTypeDecl(),
```
Comment (franklinsch) [inline lib/SymbolGraphGen/Edge.cpp:49]: Is this indentation change intentional?

**Review by franklinsch — APPROVED** [review]
Looks great!

File: lib/SymbolGraphGen/Edge.cpp (hunk 1 above)
Comment (bitjammer) [reply lib/SymbolGraphGen/Edge.cpp:49]: Nope, thanks for the catch!

**Comment by bitjammer** [comment]
@swift-ci Please smoke test

**Comment by bitjammer** [comment]
@swift-ci Please test Windows Platform

**Comment by bitjammer** [comment]
Unrelated Windows test failure.

**Pull request merged**